### PR TITLE
Add boot animation

### LINKED ISSUE #20: Boot-up screen
Make a boot-up screen that has a --Press to Start-- prompt, kinda. Really polishes this game

### PR DESCRIPTION
**Pull request by sparshg**
Added boot animation with configurable wait times between each line.
Closes #20 ?

### CODE REVIEW

File: code/main.py [hunk 1]
```diff
@@ -119,19 +119,20 @@ def run_boot():
...
+        boot_menu.run()
+        crt.draw()
+
         # Event code
         for event in pygame.event.get():
             if event.type == pygame.QUIT:
                 pygame.quit()
                 sys.exit()
-            if event.type == pygame.KEYDOWN or event.type == pygame.MOUSEBUTTONDOWN:
+            if boot_menu.boot_text.anim_completed and (event.type == pygame.KEYDOWN or event.type == pygame.MOUSEBUTTONDOWN):
```
Comment (R0merol) [inline code/main.py:131]: Players should have the ability to just skip the booting. After all, it's just like an intro cutscene. So you should leave this part of the code alone.

**Review by R0merol — CHANGES_REQUESTED** [review]
Let players skip the boot screen

**Comment by R0merol** [comment]
I was thinking if you could [add this](https://github.com/R0merol/Number-Game-v0.1/issues/20#issuecomment-1264448231) as well, I hope I'm not asking too much from you

**Comment by sparshg** [comment]
Added loading animation

**Review by R0merol — APPROVED** [review]
It's looking awesome! Thank you so much for this!

**Pull request merged**